### ISSUE TITLE
Control Panel blocked

### ISSUE THREAD

**Issue by peteralewis** [born05/craft-csp#1]
The Craft Control Panel becomes unusable without implementing unsafe inline headers (I've shortened the headers for focus):

```
'scriptSrc'     => [
    "'unsafe-inline'",
    "'unsafe-eval'"
], 
'styleSrc'       => [
    "'unsafe-inline'"
]
```

...Which isn't good for security! Am I missing something or is there another way around this?

Currently I've edited Plugin.php injecting a conditional around line 67 so it doesn't use the CSP headers if you're an admin user:

```
                if (!Craft::$app->user->getIsAdmin()) {
                    // The page is done rendering, include our headers.
                    $this->headers->setHeaders();
                }
```

Another alternative is to use:

```
Craft::$app->getRequest()->getIsCpRequest()
```

**Comment by roelvanhintum** [comment]
Are you using version 1.1.0? The plugin should not add csp headers to the cp from 1.1.0 and up.

**Comment by peteralewis** [comment]
Doh! I'd downloaded it via composer over a week ago and hadn't checked for updates since. Thanks for resolving!

**Comment by peteralewis** [comment]
Closed this thread too soon.

You need to update the composer.json with the new version number, `composer update` isn't picking up the new changes.

**Comment by roelvanhintum** [comment]
Sorry, the composer file had the wrong version. Packagist couldn't find the new version. Just released 1.1.1 to fix this. @peteralewis does this work on your end?

**Comment by peteralewis** [comment]
Awesome - thank you for the quick turnaround, all working fine.
